packages/modules/viewer-react/src/components/error/ErrorBoundary.tsx

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ErrorPage } from "@itwin/itwinui-react";
+import React, { Component } from "react";
+
+interface Props {
```

Comment:
looking really good now, i think this might be my last comment, you should be able to remove this interface, and then in line 18, you can use PropsWithChildren instead of Props

Reply:
As in PropsWithChildren<ReactNode>?

Reply:
https://github.com/iTwin/viewer/pull/173#discussion_r886111470